py/strato/common/log/log2text.py

```diff
@@ -33,17 +37,25 @@
     "\033[2;36m",
 )
 COLOR_OFF = "\033[0;0m"
+LOG_CONFIG_FILE_PATH = "/etc/stratoscale/strato-log.conf"
+HIGHEST_PRIORITY = 0
 
 class Formatter:
     _COLORS = {logging.PROGRESS: CYAN, logging.ERROR: RED, logging.WARNING: YELLOW}
 
     converter = time.gmtime
 
     def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecondPrecision, noColors, localTime=False):
+        try:
+            self.configFile = yaml.load(open(LOG_CONFIG_FILE_PATH, 'r').read())
+        except:
```

Comment:
it'll get an exception, return None, signaling that there's no format to parse the line, and it'll print it as is, with highest priority